__test__/unit/loan/loan.spec.ts

```diff
@@ -0,0 +1,24 @@
+// Internal dependencies
```

Comment:
I would also move this to a dir called integration

Reply:
Done.